src/pages/Inventory.page.js

```diff
@@ -1,15 +1,33 @@
-import { BaseSwagLabPage } from './BaseSwagLab.page';
+import { BasePage } from './Base.page';
 
-export class InventoryPage extends BaseSwagLabPage {
+export class InventoryPage extends BasePage {
     url = '/inventory.html';
 
     headerTitle = this.page.locator('.title');
 
     inventoryItems = this.page.locator('.inventory_item');
 
-    addItemToCartButton = this.page.locator('[id^="add-to-cart"]');
+    addItemToCartButton = this.page.getByTestId('[id^="add-to-cart"]');
+
+    priceElements = this.page.getByTestId('inventory-item-price');
+
+    nameElements = this.page.getByTestId('inventory-item-name');
+
+    sorting = this.page.getByTestId('product-sort-container');
 
     async addItemToCartById(id) {
         await this.addItemToCartButton.nth(id).click();
     }
+
+    async getAllPrices() {
+        return this.priceElements.allTextContents();
+    }
+
+    async getAllNames() {
+        return this.nameElements.allTextContents();
+    }
+
+    async sortingItems(option) {
```

Comment:
It's a little nitpicking, still would be much better if you name method in a way it explains (answer the question) "what this method perform" - e.g. `sortItemsBy` or `performItemsSort`